src/main/resources/view/HelpWindow.css

```
@@ -1,20 +1,17 @@
-#copyButton,
-#helpMessage {
-  -fx-text-fill: white;
-}
-
 #copyButton {
-  -fx-background-color: dimgray;
+  -fx-background-color: #F1E4C2;
+  -fx-border-color: black
+  -fx-border-radius: 10px;
 }
 
 #copyButton:hover {
-  -fx-background-color: gray;
-}
-
-#copyButton:armed {
-  -fx-background-color: darkgray;
+  -fx-background-color: #0096C9;
+  -fx-text-fill: #ffffff;
 }
 
```

Let's add a TODO for me to standardise the font.

```suggestion

/* TODO: Richard: Standardise font */
```